.gitmodules

```diff
@@ -1,4 +1,6 @@
-[submodule "vendor/cpp-driver"]
-	path = vendor/cpp-driver
+[submodule "vendor/datastax-cpp-driver"]
+	path = vendor/datastax-cpp-driver
+	url = https://github.com/datastax/cpp-driver.git
+[submodule "vendor/scylladb-cpp-driver"]
+	path = vendor/scylladb-cpp-driver
```

Comment:
do you mind if we just keep for know what we had before, the `cpp-driver` folder? but instead of using the official cpp driver using the forked version of Scylla

Reply:
by default project builds with `scylladb-cpp-driver`